modules/exploits/linux/http/traccar_rce_upload.rb

```diff
@@ -0,0 +1,262 @@
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+  include Msf::Exploit::Remote::HttpClient
```

Comment:
If you include the `FileDropper` mixin you're then able to use the handy `register_file_for_cleanup` method to clean up the cronjob after the exploit has completed.
```suggestion
  include Msf::Exploit::Remote::HttpClient
  include Msf::Exploit::FileDropper
```

Reply:
Perfect, thank you.